### PR TITLE
Tpt3 92

### PR DESCRIPTION
**Pull request by dmytro-palczewski-osi**
R-00741 rule implementation

### CODE REVIEW

File: legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/rules/ClientDroolsConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package gov.ca.cwds.cms.data.access.service.rules;
+
+import gov.ca.cwds.data.legacy.cms.entity.PlacementHome;
+import gov.ca.cwds.drools.DroolsConfiguration;
+
+/** @author CWDS CASE API Team */
```
Comment (sorych) [inline legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/rules/ClientDroolsConfiguration.java]: CASE MANAGEMENT

File: legacy-data-access-services/src/main/resources/client-legacy-rules/kmodule.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<kmodule xmlns="http://www.drools.org/xsd/kmodule">
+    <kbase name="placementHome">
```
Comment (sorych) [inline legacy-data-access-services/src/main/resources/client-legacy-rules/kmodule.xml]: why placement here?

File: legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/rules/ClientDroolsConfiguration.java (hunk 1 above)
Comment (dmytro-palczewski-osi) [reply legacy-data-access-services/src/main/java/gov/ca/cwds/cms/data/access/service/rules/ClientDroolsConfiguration.java]: fixed

**Comment by dmytro-palczewski-osi** [comment]
Tests are refactored after merging with TPT2 changes

**Pull request merged**